pyproject.toml

```diff
@@ -407,6 +414,17 @@ ignore = [  # NOTE: non-permeanent exclusions should be added to `.ruff.toml` in
 [tool.ruff.pydocstyle]
 convention = "numpy"
 
+[tool.repo-review]
+ignore = [
+    "MY",    # ignore MyPy setting checks
+    "PC111", # ignore using `blacken-docs` in pre-commit
+    "PC140", # ignore using `mypy` in pre-commit
+    "PC180", # ignore using `prettier` in pre-commit
+    "RF101", # ignore direct requiring flake8-bugbear in ruff
+    "RF102", # ignore direct requiring isort in ruff
+    "RF103", # ignore direct requiring pyupgrade in ruff
```

Comment:
We do in-fact meet these rules, but the way they are configured is not supported at this time, see scientific-python/cookie#283.

Reply:
It is supported now that [`cookie` 2023.09.21](https://github.com/scientific-python/cookie/releases/tag/2023.09.21) has been released.